src/helpFunctions/helpFunctions.js

```diff
@@ -0,0 +1,6 @@
+export function shuffledArr(arr) {
```

Comment:
It's better to not mutate array from args. Also it's better to rename to `shuffleArray`
```js
export const shuffleArray = (arr) => 
    [...arr].sort(() => Math.random() - 0.5);
```